.env

```diff
@@ -93,3 +93,5 @@ SHUFFLE_OPENSEARCH_CLOUDID=
 SHUFFLE_OPENSEARCH_PROXY=
 SHUFFLE_OPENSEARCH_INDEX_PREFIX=
 SHUFFLE_OPENSEARCH_SKIPSSL_VERIFY=true
+SHUFFLE_OPS_DASHBOARD_APIKEY=
```

Comment:
Could we automatically fill these in during the init function for elasticsearch? 

In short: if they're empty, just use a default based on what we find in the database. Health should work without any configuration from the user